finality-aleph/src/data_io/legacy/data_store.rs

```diff
@@ -261,14 +257,14 @@ where
         loop {
             self.prune_pending_messages();
             self.prune_triggers();
-
             tokio::select! {
                 maybe_message = self.messages_from_network.next() => {
                     let message = maybe_message.ok_or(Error::NetworkMessagesTerminated)?;
-                    trace!(target: "aleph-data-store", "Received message at Data Store {:?}", message);
+                    trace ! (target: "aleph-data-store", "Received message at Data Store {:?}", message);
```

Comment:
` ! `?

Reply:
Didn't even know this is parsed correctly :facepalm: Extracted log target as a bonus.